cdk-sfn-s3/bin/cdk-sfn-s3.ts

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from '@aws-cdk/core';
+import { CdkSfnS3Stack } from '../lib/cdk-sfn-s3-stack';
```

Comment:
Hey,

Is this file missing in the PR? I cant find the `cdk-sfn-s3-stack` file anywhere?

Reply:
add the missing lib folder , somehow it was with .gitignore. Thanks